content/pipeline/aws/overview.md

```diff
@@ -0,0 +1,108 @@
+---
+date: 2000-01-01T00:00:00+00:00
+title: Overview
+author: tphoney
+weight: 1
+
+related:
+  items:
+  - name: Configure Pipeline Steps
+    path: syntax/steps.md
+  - name: Configure Pipeline Services
+    path: syntax/services.md
+  - name: Integrations using Plugins
+    path: syntax/plugins.md
+
+aliases:
+- /yaml-reference/
+- /user-guide/pipeline
+
+description: |
+  Overview of AWS pipelines.
+---
+
+## Overview
+
+AWS pipelines can execute pipeline commands in two ways:
+
++ inside ephemeral Docker containers. Docker containers provide isolation, allowing safe execution of concurrent pipelines on the same machine.
++ run directly on the EC2 instance itself.
+
+Example pipeline configuration:
+
+{{< highlight text "linenos=table" >}}
+---
+kind: pipeline
+type: aws
+name: default
+
+pool:
+  use: ubuntu
+
+steps:
+- name: greeting
+  image: golang:1.12
+  commands:
+  - go build
+  - go test
+- name: echo
+  commands:
+  - ls
+...
+{{< / highlight >}}
+
+## Windows support
+
+The AWS pipelines support windows EC2 instances as a first class citizen. Syntactically they do not differ to non Windows pipelines, but the underlying infrastructure is different.
+
+{{< highlight text "linenos=table" >}}
```

Comment:
newer version of hugo support yaml highlighting

```
{{< highlight yaml "linenos=table" >}}
```

you will find this applied inconsistently in the docs because some pages were created before the feature exists, but for new pages, we can use the newer syntax.

Reply:
fixed.